.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+*/target
```

Comment:
what's the path this is trying to include, exactly? we could do just "target"  to combine the two

Reply:
Wgpu-core/target and Wgpu-types/target